src/test/java/junit/smoke/BaseTest.java

```diff
@@ -0,0 +1,32 @@
+package junit.smoke;
+
+import com.report.portal.business.steps.BaseSteps;
+import com.report.portal.business.steps.DashboardsSteps;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+import static com.report.portal.core.utils.data.RandomSequenceCreator.generateRandomInt;
+
+public class BaseTest {
+
+    protected static final String DASHBOARD_NAME = "DEMO DASHBOARD";
+    protected static final String FILTER_NAME = "DEMO_FILTER";
+    protected static final int RANDOM_VALUE = generateRandomInt(1000, 9999);
+    private final static BaseSteps baseSteps = new BaseSteps();
+    protected static DashboardsSteps dashboardsSteps;
+
+    @BeforeAll
+    public static void login() {
```

Comment:
Your test methods are just void. I suppose, BeforeAll and AfterAll methods possibly could also be static void.

Reply:
fixed!